### PR TITLE
Neuler gentx

### PR DESCRIPTION
**Pull request by neulerfzco**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Review by YazzyYaz — APPROVED** [review]

File: celestia/peers.txt [hunk 1]
```diff
@@ -1 +1 @@
-
+deefd5d3a4d497be6772295d10174f021d344ac5@57.128.75.161:26656
```
Comment (evan-forbes) [inline celestia/peers.txt:1]: please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`

**Comment by neulerfzco** [comment]
Created a new Neuler.txt with the peer info. 
I've added the blank peers.txt file but should I rather remove it instead?

**Comment by liamsi** [comment]
> I've added the blank peers.txt file but should I rather remove it instead?

Ideally, just do not modify the latest file from master.

**Comment by neulerfzco** [comment]
> Ideally, just do not modify the latest file from master.

Understood, will first sync the fork from master and re-do the process of adding both files.

**Review by liamsi — APPROVED** [review]

**Review by evan-forbes — APPROVED** [review]

**Pull request merged**